### DISCUSSION TITLE
Vorta not workin with Ventura ???

### DISCUSSION THREAD

**Discussion by MacH59-cos** [borgbase/vorta#1524] [General]
Am I mistaken or does Vorta still not work on a Mac with Ventura? I read somewhere that all qt-based apps crash on Ventura. Is that the reason and how can I fix that?

**Comment by m3nu** [comment]
Always worked fine on Ventura and didn't hear anything similar.

If you get a specific error, it will help sharing that.

![Screenshot 2022-12-28 at 18 46 02](https://user-images.githubusercontent.com/3916435/209852326-fe40272f-2b4d-41e2-bc9d-599e40ee0e91.jpg)
